packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
 # Unreleased
+- [breaking] `QuerySnapshot.docChanges()` is now a method. Please make sure
+  to audit your code for all existing usage of the old property `.docChanges`
+  and to change these to method invocations.
+- [changed] Replaced `QueryListenOptions` and `DocumentListenOptions` with
+  `SnapshotListenOptions`. `SnapshotListenOptions` only exposes the single
+  property `includeMetadataChanges` to turn on metadata changes.
+- [changed] `QuerySnapshot.docChanges()` optionally takes
+  `SnapshotListenOptions`. By default, even when listening to a query with
```

Comment:
Similarly, I'd say "'... optionally takes a `includeMetadataChanges` boolean option. By default, ..."

Also, can we either merge this with the top docChanges() change or at least put them next to each other?  Given your 3 entries right now, perhaps the optimal order would be 2,1,3.

Reply:
I merged this into two bullet points and moved the `docChanges` one further down. I wanted to emphasize the breaking change in the original changelog entry.